app/scaffold-www/welcome-view/welcome.html

```diff
@@ -0,0 +1,33 @@
+<html>
+    <head>
+        <meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=1, user-scalable=no">
+        <script src="../astro-client.js"></script>
+        <link href='https://fonts.googleapis.com/css?family=Lato:400,700,900' rel='stylesheet' type='text/css'>
```

Comment:
Is this a web font import? Would it be hard to import the font into the Scaffold? I think we can distribute it freely.